app/extract_translation_data.py

```diff
@@ -69,6 +72,105 @@ def get_text(data):
     return translatable_text
 
 
+def get_non_question_translatable_text(container):
+    extracted_text = []
+    if container['type'] == 'Introduction':
+        extracted_text.extend(get_introduction_translatable_text(container))
+    elif container['type'] == 'Interstitial':
+        extracted_text.extend(get_interstitial_translatable_text(container))
+
+    return extracted_text
```

Comment:
Bikeshedding, but if the number of types was longer I often use a pattern along the lines of:

```
TYPE_FUNC = {
    'Introduction': get_introduction_translatable_text
    'Interstitial': get_interstitial_translatable_text
}
```

```
return TYPE_FUNC.get(container['type'], lambda x: [])(container)
```